jupyterlab/handlers/yjs_echo_ws.py

```diff
@@ -0,0 +1,79 @@
+import uuid
+import json
```

Comment:
There are three code scanning check notices in this module. I do not know if everyone sees them, but in case if those do not display for you, those are:

- Import of 'json' is not used.
- Testing equality to None (x2)

Reply:
Thanks, fixed in https://github.com/jupyter-rtc/jupyterlab/pull/74